datasetinsights/evaluation_metrics/average_recall_2d.py

```diff
@@ -111,7 +111,7 @@ class MeanAverageRecallAverageOverIOU(EvaluationMetric):
     TYPE = "scalar"
 
     IOU_THRESHOULDS = np.linspace(
-        0.5, 0.95, np.round((0.95 - 0.5) / 0.05) + 1, endpoint=True
+        0.5, 0.95, int(np.round((0.95 - 0.5) / 0.05)) + 1, endpoint=True
```

Comment:
Is this the issue that was fixed in numpy 1.18+

Reply:
yup. linspace expects an int but round returns a numpy float64. i'm guessing that used to get converted to an int before, but it's probably not a good practice to rely on implicit type conversion either way.